### PR TITLE
Docs(repo): Introduce form validations package in main README file

### PR DESCRIPTION
**Pull request by literat**
## Description

Add missing package to the table with provided packages in main readme.

### Issue reference

---

### Before submitting the PR, please make sure you do the following

- [x] Read the [Contributing Guidelines](https://github.com/lmc-eu/spirit-desing-system/blob/main/CONTRIBUTING.md).
- [x] Follow the [PR Title/Commit Message Convention](https://github.com/lmc-eu/spirit-desing-system/blob/main/CONTRIBUTING.md#commit-conventions).
- [x] Provide a description in this PR that addresses **what** the PR is solving, or reference the issue that it solves (e.g. `fixes #123`).
- [x] Ideally, include relevant tests that fail without this PR but pass with it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,3 +58,5 @@ See the [LICENSE](LICENSE.md) file for information.
 [swr-badge]: https://img.shields.io/npm/v/%40lmc-eu/spirit-web-react.svg?style=flat-square
 [swt-packagist]: https://packagist.org/packages/lmc/spirit-web-twig-bundle
 [swt-badge]: https://img.shields.io/packagist/v/lmc/spirit-web-twig-bundle.svg?style=flat-square
+[sfv-packagist]: https://packagist.org/packages/lmc/spirit-form-validations
+[sfv-badge]: https://img.shields.io/packagist/v/lmc/spirit-form-validations.svg?style=flat-square
```
Comment (crishpeen) [inline README.md:62]:
```suggestion
[sfv-badge]: https://img.shields.io/npm/v/%40lmc-eu/spirit-form-validations.svg?style=flat-square
```

File: README.md [hunk 2]
```diff
@@ -57,3 +58,5 @@ See the [LICENSE](LICENSE.md) file for information.
 [swr-badge]: https://img.shields.io/npm/v/%40lmc-eu/spirit-web-react.svg?style=flat-square
 [swt-packagist]: https://packagist.org/packages/lmc/spirit-web-twig-bundle
 [swt-badge]: https://img.shields.io/packagist/v/lmc/spirit-web-twig-bundle.svg?style=flat-square
+[sfv-packagist]: https://packagist.org/packages/lmc/spirit-form-validations
```
Comment (crishpeen) [inline README.md:61]:
```suggestion
[sfv-npm]: https://www.npmjs.com/package/@lmc-eu/spirit-form-validations
```

**Review by crishpeen — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (literat) [reply README.md]: 🤦  Fixed.

File: README.md (hunk 2 above)
Comment (literat) [reply README.md]: 🤦 Fixed.

**Review by crishpeen — APPROVED** [review]

**Pull request merged**